README.md

```diff
@@ -175,3 +175,51 @@ $ zcat ~/repos/rhmi-utils/samples/xyz.json.gz | jq -f scripts/resources_by_names
 "ops-health-monitoring",,,,,,,
 ```
 
+## Using `ocm` for installation of RHMI
+
+If you want to test your changes on a cluster, the easiest solution would be to spin up OSD 4 cluster using `ocm`. If you want to spin up a cluster using BYOC (your own AWS credentials), follow the additional steps marked as **BYOC**.
+
+#### Prerequisites
+* [OCM CLI](https://github.com/openshift-online/ocm-cli/releases)
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+* [jq](https://stedolan.github.io/jq/)
+
+#### Steps
+
+1. Download the CLI tool and add it to your PATH
+2. Export [OCM_TOKEN](https://github.com/openshift-online/ocm-cli#log-in): `export OCM_TOKEN="<TOKEN_VALUE>"`
+3. Login via OCM: 
+```
+make ocm/login
+```
+
+**BYOC**
+Make sure you have credentials for IAM user with admin access to AWS and other IAM user called "osdCcsAdmin" created in AWS, also with admin access.
+Export the credentials for your IAM user, set BYOC variable to `true` and create a new access key for "osdCcsAdmin" user:
+```
+export AWS_ACCOUNT_ID=<REPLACE_ME>
+export AWS_ACCESS_KEY_ID=<REPLACE_ME>
+export AWS_SECRET_ACCESS_KEY=<REPLACE_ME>
+export BYOC=true
+make ocm/aws/create_access_key
+```
+
+4. Create cluster template: `make ocm/cluster.json`.
+
+This command will generate `ocm/cluster.json` file with generated cluster name. This file will be used as a template to create your cluster via OCM CLI.
+By default, it will set the expiration timestamp for a cluster for 4 hours, meaning your cluster will be automatically deleted after 4 hours after you generated this template. If you want to change the default timestamp, you can update it in `ocm/cluster.json` or delete the whole line from the file if you don't want your cluster to be deleted automatically at all. 
+
+5. Create the cluster: `make ocm/cluster/create`.
+
+This command will send a request to [Red Hat OpenShift Cluster Manager](https://cloud.redhat.com/) to spin up your cluster and waits until it's ready. You can see the details of your cluster in `ocm/cluster-details.json` file
+
+6. Once your cluster is ready, OpenShift Console URL will be printed out together with the `kubeadmin` user & password. These are also saved to `ocm/cluster-credentials.json` file. Also there will be `ocm/cluster.kubeconfig` file created that you can use for running `oc` commands right away, for example, for listing all projects on your OpenShift cluster:
+
+```
+oc --config ocm/cluster.kubeconfig projects
+```
+
+7. If you want to install the latest released RHMI, you can trigger it by applying an RHMI addon.
+Run `make ocm/install/rhmi-addon` to trigger the installation. Once the installation is completed, the installation CR with RHMI components info will be printed to the console.
```

Comment:
I don't see a make command with this name `ocm/install/rhmi-addon`

Reply:
The command is [here](https://github.com/integr8ly/rhmi-utils/pull/20/files#diff-c90e3921bdbf84a8bb9bb3e7179702e5R35-R37)
I talked to Mike about the structure of the scripts, and we agreed that the best solution will be to keep it the same as in integreatly-operator repo, i.e. put all the ocm-related targets into a separate .mk file and import it from the [Makefile in root directory](https://github.com/integr8ly/rhmi-utils/pull/20/files#diff-b67911656ef5d18c4ae36cb6741b7965R1)

Reply:
Apologies, I missed this. Browser search doesn't work with a `viewed` file in a PR :-)